packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -121,10 +127,26 @@ Spec.prototype.onException = function onException(e) {
     return;
   }
 
+  const {expected, actual} = e;
+  let message;
+  if (expected && actual) {
+    const diffString = diff(expected, actual, {
+      expand: this.expand,
+    });
+    message = matcherHint('.toBe') +
+      '\n\n' +
+      `Expected value to be (using ===):\n` +
+      `  ${printExpected(expected)}\n` +
+      `Received:\n` +
+      `  ${printReceived(actual)}` +
+      (diffString ? `\n\nDifference:\n\n${diffString}` : '');
+  }
+
   this.addExpectationResult(
     false,
     {
-      matcherName: '',
+      matcherName: 'blah blah',
```

Comment:
JKJK, when the `matcherName` is changed, in addition to when the `message` is not `undefined` a bunch of snapshots fail because messages are better now:

<img width="701" alt="screen shot 2017-03-28 at 7 25 28 am" src="https://cloud.githubusercontent.com/assets/1500684/24407330/c727a6b4-1387-11e7-8b2c-9e1072638a98.png">

<img width="833" alt="screen shot 2017-03-28 at 7 25 40 am" src="https://cloud.githubusercontent.com/assets/1500684/24407332/c9e16f02-1387-11e7-9dbb-a09fbcd9b133.png">